public/src/main/java/com/revenuecat/purchases/PurchaserInfo.kt

```diff
@@ -0,0 +1,190 @@
+//  Purchases
+//
+//  Copyright © 2019 RevenueCat, Inc. All rights reserved.
+//
+
+package com.revenuecat.purchases
+
+import android.net.Uri
+import android.os.Parcelable
+import com.revenuecat.purchases.models.Transaction
+import com.revenuecat.purchases.parceler.JSONObjectParceler
+import kotlinx.android.parcel.IgnoredOnParcel
+import kotlinx.android.parcel.Parcelize
+import kotlinx.android.parcel.TypeParceler
+import org.json.JSONObject
+import java.util.Date
+
+/**
+ * Class containing all information regarding the purchaser
+ * @property entitlements Entitlements attached to this purchaser info
+ * @property purchasedNonSubscriptionSkus Set of non-subscription, non-consumed skus
+ * @property allExpirationDatesByProduct Map of skus to expiration dates
+ * @property allPurchaseDatesByProduct Map of skus to purchase dates
+ * @property requestDate Date when this info was requested
+ * @property firstSeen The date this user was first seen in RevenueCat.
+ * @property originalAppUserId The original App User Id recorded for this user.
+ * @property managementURL URL to manage the active subscription of the user. If this user has an active iOS
+ * subscription, this will point to the App Store, if the user has an active Play Store subscription
+ * it will point there. If there are no active subscriptions it will be null.
+ * If there are multiple for different platforms, it will point to the Play Store
+ * @property originalPurchaseDate the purchase date for the version of the application when the user bought the app.
+ * Use this for grandfathering users when migrating to subscriptions. This can be null, see -Purchases.restorePurchases
+ */
+@Parcelize
+@TypeParceler<JSONObject, JSONObjectParceler>()
+@Deprecated(
+    "Renamed to CustomerInfo",
+    replaceWith = ReplaceWith(expression = "CustomerInfo")
+)
+data class PurchaserInfo constructor(
```

Comment:
does this one not need the -jvmname thing?

Reply:
I didn't added it because this is deprecated with `WARNING` level (the default). I only added the `jvmname` thing to the ones I deprecated with `ERROR`. `WARNING` and `ERROR` look the same in Java, so I added the jvmname to break the compilation in the `ERROR` cases. 

Anyone accessing the `PurchaserInfo` in Java will most likely have other compilation issues, like if they are assigning the result of `getCustomerInfo` to a `PurchaserInfo`, their app won't compile anyhow, so I figured it should be enough by doing a `WARNING`